src/components/Projects.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+    <div>
+        <div class="Project">
+            <h1 class="projectTitle">Weather Station</h1>
+            <p class="projectText">This is our <a href="https://weather.codihacks.org/" target="_blank">weather station</a> web app project! It features a physical weather station, an API server
+                written in Rust, and a web app frontend written in Vue.
+            </p>
+        </div> 
+        <div class="Project">
+            <h1 class="projectTitle">Space Race</h1>
+            <p class="projectText">This is our Spaceship Race project! It is a spaceship game created with the Lua framework <a href="https://love2d.org/" target="_blank">Love2D</a>.
+                It lacks levels, so if you are interested in helping, please open an issue on our GitHub.
+            </p>
+            
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: 'Projects',
+    props: {
+    darkMode: {
+      type: Boolean,
+      required: true
+    }
+  }
+
+}
+</script>
```

Comment:
<script> block isn't necessary for components with only static content. `darkMode` prop isn't necessary either,